RHEL/6/input/oval/grub_enable_fips_mode.xml

```diff
@@ -0,0 +1,34 @@
+<def-group>
+  <definition class="compliance" id="grub_enable_fips_mode" version="1">
+    <metadata>
+      <title>Enable FIPS Mode in GRUB</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 6</platform>
+      </affected>
+      <description>Look for argument fips=1 in the kernel line in /etc/grub.conf.</description>
+      <reference source="galford" ref_id="20160608" ref_url="test_attestation" />
+    </metadata>
+    <criteria operator="AND">
+      <extend_definition comment="Installed OS is RHEL6" definition_ref="installed_OS_is_rhel6" />
+      <criterion test_ref="test_grub_enable_fips_mode" comment="check for fips=1 in /etc/grub.conf" />
```

Comment:
Also, shouldn't it depend on dracut-fips RPM being installed? (together with prelinking disabled)